pkg/apis/v1beta1/ec2nodeclass.go

```diff
@@ -356,7 +355,7 @@ func (in *EC2NodeClass) InstanceProfileRole() string {
 func (in *EC2NodeClass) InstanceProfileTags(clusterName string) map[string]string {
 	return lo.Assign(in.Spec.Tags, map[string]string{
 		fmt.Sprintf("kubernetes.io/cluster/%s", clusterName): "owned",
-		corev1beta1.ManagedByAnnotationKey:                   clusterName,
+		EksClusterNameAnnotationKey:                          clusterName,
```

Comment:
nit: follow the Go semantic of capitalizing the acronym
```suggestion
		EKSClusterNameAnnotationKey:                          clusterName,
```